airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/default_error_handler.py

```diff
@@ -92,7 +92,6 @@ class DefaultErrorHandler(ErrorHandler, JsonSchemaMixin):
 
     DEFAULT_BACKOFF_STRATEGY = ExponentialBackoffStrategy
 
-    config: Config
```

Comment:
noticed that we define config twice. not related to the refactor, but just fixing things i find